--- warsow_1.02_sdk/source/Makefile.orig	2013-04-15 00:50:05.337871495 +0200
+++ warsow_1.02_sdk/source/Makefile	2013-04-15 00:51:13.270701812 +0200
@@ -210,7 +210,7 @@
 CFLAGS_TV_SERVER=-DDEDICATED_ONLY -DTV_SERVER_ONLY -DC_ONLY -DTV_MODULE_HARD_LINKED
 
 LDFLAGS_COMMON=$(LDFLAGS)
-LXXFLAGS_COMMON=$(LDFLAGS) -lstdc++ -lsupc++
+LXXFLAGS_COMMON=$(LDFLAGS) -lstdc++
 
 ifeq ($(OS),FreeBSD)
 LDFLAGS_COMMON+= -L/usr/local/lib -lm -pthread
@@ -218,11 +218,6 @@
 LDFLAGS_COMMON+= -L/usr/local/lib -ldl -lm -O1 -Wl,--as-needed
 endif
 
-ifneq ($(DEBUG_BUILD),YES)
-LDFLAGS_COMMON+= -s
-LXXFLAGS_COMMON+= -s
-endif
-
 ifeq ($(ARCH),x86_64)
 LIB=lib64
 else
@@ -286,7 +281,7 @@
 LDFLAGS_CLIENT=-lz -lcurl -framework libjpeg -framework SDL -framework OpenGL -I/Library/Frameworks/SDL.framework/Headers
 LDFLAGS_MODULE=-bundle
 LDFLAGS_COMMON=-arch ppc -arch i386 -framework AppKit -mmacosx-version-min=10.4 -isysroot /Developer/SDKs/MacOSX10.4u.sdk
-LXXFLAGS_COMMON=$(LDFLAGS_COMMON) -lstdc++ -lsupc++
+LXXFLAGS_COMMON=$(LDFLAGS_COMMON) -lstdc++
 LDFLAGS_DED=-lz -lcurl
 LDFLAGS_QF=-framework SDL -framework Ogg -framework Vorbis
 LDFLAGS_OPENAL=-framework OpenAL -framework Ogg -framework Vorbis
